packages/babel-helpers/src/helpers/applyDecs.js

```diff
@@ -699,23 +696,19 @@ export default function applyDecs(targetClass, memberDecs, classDecs) {
   var ret = [];
   var staticMetadataMap = {};
 
-  if (memberDecs) {
```

Comment:
The guard is removed because the transformer always provide an array literal for `memberDecs`